web_src/js/features/comp/LabelEdit.js

```diff
@@ -6,23 +6,23 @@ function isExclusiveScopeName(name) {
 }
 
 function updateExclusiveLabelEdit(form) {
-  const $nameInput = $(`${form} .label-name-input`);
-  const $exclusiveField = $(`${form} .label-exclusive-input-field`);
-  const $exclusiveCheckbox = $(`${form} .label-exclusive-input`);
-  const $exclusiveWarning = $(`${form} .label-exclusive-warning`);
+  const nameInput = document.querySelector(`${form} .label-name-input`);
+  const exclusiveField = document.querySelector(`${form} .label-exclusive-input-field`);
+  const exclusiveCheckbox = document.querySelector(`${form} .label-exclusive-input`);
+  const exclusiveWarning = document.querySelector(`${form} .label-exclusive-warning`);
 
-  if (isExclusiveScopeName($nameInput.val())) {
-    $exclusiveField.removeClass('muted');
-    $exclusiveField.removeAttr('aria-disabled');
-    if ($exclusiveCheckbox[0].checked && $exclusiveCheckbox.data('exclusive-warn')) {
-      $exclusiveWarning.removeClass('tw-hidden');
+  if (isExclusiveScopeName(nameInput.value)) {
+    exclusiveField?.classList.remove('muted');
+    exclusiveField?.removeAttribute('aria-disabled');
+    if (exclusiveCheckbox.checked && exclusiveCheckbox.getAttribute('data-exclusive-warn')) {
```

Comment:
Here is a regression bug, the code below uses `$exclusiveCheckbox.data('exclusive-warn', ...` and can't be read by `getAttribute`

Reply:
`.data("foo")` is equivalent to `.getAttribute("data-foo")`, no?

Reply:
`getAttribute('data-foo')` can't read `$().data('foo', 'value')`, see below `$exclusiveCheckbox.data('exclusive-warn',...)`

Reply:
Ah indeed. I forgot that `.data()` does not actually write HTML attributes. Should have refactored the whole file.